operator/v1/types_ingress.go

```diff
@@ -0,0 +1,119 @@
+package v1
```

Comment:
Be sure to set `+optional` on everything that is optional.

Reply:
Done